test/unit/commands.test.ts

```diff
@@ -41,104 +42,229 @@ const nKeyWithInvalidUtf8 =
 const msgBodyNKeyWithInvalidUtf8 = Buffer.from(nKeyWithInvalidUtf8, 'hex');
 
 describe('BinMsg BSON utf8 validation', () => {
-  it('bson correctly deserializes data with replacement characters for invalid utf8 in writeErrors object', () => {
-    // this is a sanity check to make sure nothing unexpected is happening in the deserialize method itself
+  describe('when bson-ext is installed', () => {
+    before(function () {
+      if (!isBSONExtImported()) {
+        this.skip();
```

Comment:
Can we attach a skip reason?  This comment also applies to the js-bson section of the tests.

Reply:
sure!